test/system/220-healthcheck.bats

```diff
@@ -487,13 +488,20 @@ function _check_health_log {
     rc=0
     wait -n $hc_pid || rc=$?
     assert $rc -eq 1 "exit status check of healthcheck command"
+    assert $(cat $hcStatus) =~ "stopped" "Health status"
```

Comment:
`"$(< $hcStatus)"`, cat is not needed. It Would also be good if you could do a full string match `==` here to avoid any extra error lines printed